packages/babel/types/index.d.ts

```diff
@@ -0,0 +1,130 @@
+   * @default undefined;
+   */
+  exclude?: FilterPattern;
+  /**
+   * An array of file extensions that Babel should transpile. If you want to transpile TypeScript files with this plugin it's essential to include .ts and .tsx in this option.
+   * @default ['.js', '.jsx', '.es6', '.es', '.mjs']
+   */
+  extensions?: string[];
+  /**
+   * A boolean value indicating whether to bundle in the Babel helpers.
+   * @default false
+   */
+  externalHelpers?: boolean;
+  /**
+   * An array which gives explicit control over which babelHelper functions are allowed in the bundle.
+   * @default undefined
+   */
+  externalHelpersWhitelist?: string[];
+  /**
+   * @default false
+   */
+  runtimeHelpers?: boolean;
+  /**
+   * Use with other formats than UMD/IIFE.
+   * @default false
+   */
+  allowAllFormats?: boolean;
+}
+
+type CustomInputPluginOptions = (
+  options: RollupBabelInputPluginOptions & Record<string, any>
+) => {
+  customOptions: Record<string, any>;
+  pluginOptions: RollupBabelInputPluginOptions;
+};
+type CustomOutputPluginOptions = (
+  options: RollupBabelOutputPluginOptions & Record<string, any>
+) => {
+  customOptions: Record<string, any>;
+  pluginOptions: RollupBabelOutputPluginOptions;
+};
+type CustomPluginConfig = (
+  cfg: babelCore.PartialConfig,
+  options: { code: string; customOptions: Record<string, any> }
+) => babelCore.TransformOptions;
+type CustomPluginResult = (
+  result: babelCore.BabelFileResult,
+  options: {
+    code: string;
+    customOptions: Record<string, any>;
+    config: babelCore.PartialConfig;
+    transformOptions: babelCore.TransformOptions;
+  }
+) => babelCore.BabelFileResult;
+interface CustomInputPlugin {
+  options?: CustomInputPluginOptions;
+  config?: CustomPluginConfig;
+  result?: CustomPluginResult;
+}
+interface CustomOutputPlugin {
+  options?: CustomOutputPluginOptions;
+  config?: CustomPluginConfig;
+  result?: CustomPluginResult;
+}
+type CustomInputPluginBuilder = (babel: typeof babelCore) => CustomInputPlugin;
+type CustomOutputPluginBuilder = (babel: typeof babelCore) => CustomOutputPlugin;
+
+/**
+ * A Rollup plugin for seamless integration between Rollup and Babel.
+ * @param options - Plugin options.
+ * @returns Plugin instance.
+ */
+export function getBabelInputPlugin(options?: RollupBabelInputPluginOptions): Plugin;
+export function getBabelOutputPlugin(options?: RollupBabelOutputPluginOptions): Plugin;
+
+export function createBabelInputPluginFactory(
+  customCallback?: CustomInputPluginBuilder
+): typeof getBabelInputPlugin;
+export function createBabelOutputPluginFactory(
+  customCallback?: CustomOutputPluginBuilder
+): typeof getBabelOutputPlugin;
+
+/**
+ * A Rollup plugin for seamless integration between Rollup and Babel.
+ * @param options - Plugin options.
+ * @returns Plugin instance.
+ */
+export default function babel(options?: RollupBabelInputPluginOptions): Plugin;
```

Comment:
`babel` is also a named export

```suggestion
export function babel(options?: RollupBabelInputPluginOptions): Plugin;
export default babel;
```